ja/contents.rst

```diff
@@ -74,10 +74,10 @@
     :maxdepth: 3
     :caption: プラグイン
 
-    authorization <https://book.cakephp.org/authorization/>
-    authentication <https://book.cakephp.org/authentication/>
+    authorization </authorization/>
+    authentication </authentication/>
     chronos
-    debug-kit
+    debug-kit </debugkit/>
```

Comment:
```suggestion
    debug-kit </debugkit/3.x/jp/>
```

Should it be `ja`?

As I just learned, `jp` is country code, `ja` is language code.

Reply:
It should be `ja` I'll fix the debug kit code.